apm-sniffer/apm-agent/src/main/java/org/apache/skywalking/apm/agent/SkyWalkingAgent.java

```diff
@@ -130,6 +130,8 @@ public static void premain(String agentArgs, Instrumentation instrumentation) th
                     .with(new Listener())
                     .installOn(instrumentation);
 
+        SnifferConfigInitializer.pluginInitCompleted();
```

Comment:
```suggestion
        PluginFinder.pluginInitCompleted();
```

Reply:
I think `pluginInitCompleted` show belong to `SnifferConfigInitializer`.

the IS_PLUGIN_INIT_COMPLETED field is not a flag that indicates the plugins are loaded. It's the flag that ByteBuddy has loaded all plugins. Before this, ThreadPoolExecutor can't be created.

Reply:
No, you have a misunderstanding with the codes. PluginFinder is the one working with bytebuddy. SnifferConfigInitializer is just about `Config.class`.

Please read the codes again.

Reply:
Putting it in the Agent makes sense.

Reply:
> Member

Ok, Then keep it like this.

Reply:
I think you still don't follow all requirements. Rename is required, and move to `SkyWalkingAgent#IS_PLUGIN_INSTALLED_COMPLETED`.



> > Putting it in the Agent makes sense.
> 
> Ok, Then keep it like this.

I think I said putting it in the Agent class, not the way it is right now. You somehow misunderstood the point.

Reply:
> Please move and rename this flag, I think others are good to me.

Ok, I understand it.  `SnifferConfigInitializer` is just responsible for configuration.  I will move it to the `PluginFinder`.